client/src/App.js

```diff
@@ -9,18 +9,19 @@ import {
   useLocation,
 } from "react-router-dom";
 import React, { useState, useEffect } from 'react';
+import GoogleMapReact from 'google-map-react';
 
 const splashPads = [
-  { path: '/bartholomew', title: 'Bartholomew Park Splashpad', parkKey: 'bartholomew' },
-  { path: '/chestnut', title: 'Chestnut', parkKey: 'chestnut' },
-  { path: '/eastwoods', title: 'Eastwoods', parkKey: 'eastwoods' },
-  { path: '/liz-carpenter', title: 'Liz Carpenter Park', parkKey: 'liz-carpenter' },
-  { path: '/lott', title: 'Lott', parkKey: 'lott' },
-  { path: '/metz', title: 'Metz', parkKey: 'metz' },
-  { path: '/mueller-branch-park', title: 'Mary Elizabeth Branch Park', parkKey: 'mueller-branch-park' },
-  { path: '/pease', title: 'Pease Park', parkKey: 'pease' },
-  { path: '/ricky-guerrero', title: 'Ricky Guerrero Park', parkKey: 'ricky-guerrero' },
-  { path: '/rosewood', title: 'Rosewood Park', parkKey: 'rosewood' },
+  { path: '/bartholomew', title: 'Bartholomew Park Splashpad', parkKey: 'bartholomew', lat: 30.305177, lng: -97.697469 },
+  { path: '/chestnut', title: 'Chestnut', parkKey: 'chestnut', lat: 30.277067251010195, lng: -97.71699243984315 },
+  { path: '/eastwoods', title: 'Eastwoods', parkKey: 'eastwoods', lat: 30.290943, lng: -97.731715 },
+  { path: '/liz-carpenter', title: 'Liz Carpenter Park', parkKey: 'liz-carpenter', lat: 30.26211031582992, lng: -97.75441430399374 },
+  { path: '/lott', title: 'Lott', parkKey: 'lott', lat: 30.271041206071896, lng: -97.7297671152558 },
+  { path: '/metz', title: 'Metz', parkKey: 'metz', lat: 30.252258120681006, lng: -97.71810034083994 },
+  { path: '/mueller-branch-park', title: 'Mary Elizabeth Branch Park', parkKey: 'mueller-branch-park', lat: 30.300190, lng: -97.702812 },
+  { path: '/ricky-guerrero', title: 'Ricky Guerrero Park', parkKey: 'ricky-guerrero', lat: 30.247392178325605, lng: -97.76433234845857 },
+  { path: '/rosewood', title: 'Rosewood Park', parkKey: 'rosewood', lat: 30.27109617907063, lng: -97.71390894361079 },
+  { path: '/pease', title: 'Pease Park', parkKey: 'pease', lat: 30.281754288606752, lng: -97.7521610530982 },
```

Comment:
Should we get these from the server?

Reply:
yeah, I'll probably clean this up. I wanted to test the map first.